features/emscripten/build.bat

```diff
@@ -1,4 +1,6 @@
 @echo off
+set CONAN_USER_HOME=%USERPROFILE%
```

Comment:
Windows Max Path :trollface:

Reply:
Moved to `C:/projects/CONAN_HOME/<example-name>`

Reply:
Are you keeping this? Is it not possible to use the `C:/projects/CONAN_HOME/<example-name>` path?

Reply:
done